ucdavis_service/src/main/java/com/ucdavis/application/WebController/ServiceController.java

```diff
@@ -66,6 +66,7 @@ public String submit(@Valid @ModelAttribute("request") final Request request,fin
 		List<JSONObject> list = parser.getListFromJson(parser.extractDataItems((parser.jsonStringToMap(jsonObject)).get("Records").toString()));
 		// write map
 		davisService.writeToMap(list, queryMapping.get("targets"));
+		davisService.writeToCSV(queryMapping.get("targets"));
```

Comment:
Please check if write to CSV from map is also working

Reply:
WriteToMap is based on the map null-checked, it is working now.

Reply:
WriteToCSV is based on the HashMap after null check. It has been tested.